compose/cli/main.py

```diff
@@ -485,8 +487,7 @@ def handler(signal, frame):
                 signal.signal(signal.SIGINT, handler)
 
                 print("Gracefully stopping... (press Ctrl+C again to force)")
-                timeout = options.get('--timeout')
-                params = {} if timeout is None else {'timeout': int(timeout)}
+                params = {} if timeout is None else {'timeout': timeout}
                 project.stop(service_names=service_names, **params)
```

Comment:
Ah right, because `docker-py` specifies `timeout=10` instead of `timeout=None`. That's unfortunate.